src/commonMain/kotlin/Main.kt

```diff
@@ -1,30 +1,54 @@
+import komem.litmus.RunParams
 import komem.litmus.barriers.CinteropSpinBarrier
-import komem.litmus.mergeOutcomes
+import komem.litmus.calcStats
+import komem.litmus.litmusTest
 import komem.litmus.prettyPrint
+import komem.litmus.runners.LitmusTestRunner
 import komem.litmus.runners.WorkerTestRunner
-import komem.litmus.runners.runTestParallel
-import komem.litmus.tests.SBTest
-import komem.litmus.variateRunParams
-import kotlin.time.Duration.Companion.seconds
+import kotlin.time.measureTime
 
 fun main() {
-    val runner = WorkerTestRunner
 
-    val paramsList = variateRunParams(
-        batchSizeSchedule = listOf(100_000),
-        affinityMapSchedule = listOf(null), // getAffinityManager()?.presetShort() ?: listOf(null),
-        syncPeriodSchedule = generateSequence(3) { it * 4 }.take(3).toList(),
-        memShufflerProducerSchedule = listOf(null),
-        barrierSchedule = listOf(::CinteropSpinBarrier)
-    ).toList()
+    class Data {
+        var x = 0
+        var y = 0
+        var a = 0
+        var b = 0
+    }
 
-    val singleTestDuration = 3.seconds
-    val testProducer = ::SBTest
+    val sb = litmusTest(::Data) {
+        thread {
+            x = 1
+            a = y
+        }
+        thread {
+            y = 1
+            b = x
+        }
+        outcome {
+            listOf(a, b)
+        }
+        spec {
+            accepted = setOf(
+                listOf(1, 1), listOf(1, 0), listOf(0, 1)
```

Comment:
Why `List` instead of `Pair` ?

Reply:
For consistency with tests that have more than two values as outcome